cairo/tests/utils/strategies.py

```diff
@@ -317,7 +317,9 @@ def tuple_strategy(thing):
 
 
 # Fork
-state = st.lists(address, min_size=0, max_size=MAX_ADDRESS_SET_SIZE).flatmap(
+state = st.lists(
+    address, min_size=0, max_size=MAX_ADDRESS_SET_SIZE, unique=True
```

Comment:
Nice catch 
Unique is really important indeed
```suggestion
    address, max_size=MAX_ADDRESS_SET_SIZE, unique=True
```
By default min_size = 0